server/src/main/java/com/defold/extender/services/GradleService.java

```diff
@@ -138,16 +139,17 @@ public long getCacheSize() throws IOException {
     ///////////////////////////////////////////////////////////////////////////////////////////////
     // Private
 
-    private void createGradlePropertiesFile(File gradlePropertiesFile, Boolean useJetifier) throws IOException {
+    private void createGradlePropertiesFile(File gradlePropertiesFile, Boolean useJetifier, Map<String, Object> jobEnvContext) throws IOException {
         HashMap<String, Object> envContext = new HashMap<>();
         envContext.put("android-enable-jetifier", useJetifier.toString());
+        envContext.put("org-gradle-jave-home", GRADLE_JAVA_HOME);
         String contents = templateExecutor.execute(gradlePropertiesTemplateContents, envContext);
         Files.write(gradlePropertiesFile.toPath(), contents.getBytes());
     }
 
     private void createLocalPropertiesFile(File localPropertiesFile, Map<String, Object> jobEnvContext) throws IOException {
         HashMap<String, Object> envContext = new HashMap<>();
-        envContext.put("android-sdk-root", jobEnvContext.get("env.ANDROID_SDK_ROOT"));
+        envContext.put("sdk-dir", jobEnvContext.get("env.ANDROID_HOME"));
```

Comment:
I'd rather have "android-home" instead?
Better for readability/discoverability imho.